geode-lucene/src/main/java/org/apache/geode/cache/lucene/internal/LuceneServiceImpl.java

```diff
@@ -298,9 +259,9 @@ protected boolean createLuceneIndexOnDataRegion(final PartitionedRegion userRegi
              * which is responsible for indexing the user region.
              *
              **/
-            repositoryManager.getRepository(primaryBucketId);
+            repositoryManager.computeRepository(primaryBucketId);
```

Comment:
I'm not sure why this is changed as part of this refactoring?

Reply:
ahhh!! sorry barry!! this got checked in by mistake.. we were experimenting something and this missed the revert